modules/openapi-generator/src/main/resources/dart2/serialization/native/native_class.mustache

```diff
@@ -1,21 +1,40 @@
 class {{{classname}}} {
 {{>dart_constructor}}
-
-  {{#vars}}
+{{#vars}}
   {{#description}}
   /// {{{.}}}
   {{/description}}
   {{^isEnum}}
     {{#minimum}}
-  // minimum: {{{.}}}
+      {{#description}}
+  ///
+      {{/description}}
+  /// Minimum value: {{{.}}}
     {{/minimum}}
     {{#maximum}}
-  // maximum: {{{.}}}
+      {{#description}}
+        {{^minimum}}
+  ///
+        {{/minimum}}
+      {{/description}}
+  /// Maximum value: {{{.}}}
     {{/maximum}}
+    {{^isNullable}}
+      {{^required}}
+        {{^defaultValue}}
+  ///
+  /// Please note: This property should have been non-nullable! Since the specification file
+  /// does not include a default value (using the "default:" property), however, the generated
+  /// source code must fall back to having a nullable type.
+  /// Consider adding a "default:" property in the specification file to hide this note.
+  ///
```

Comment:
Since the template is enforcing a nullable property even though it's supposed to be non-nullable, it's best to notify the developer about this incident. In most cases, this is a bug waiting to happen so I think it's best to have this notice.